pdir/attr_category.py

```diff
@@ -215,27 +231,27 @@ def is_descriptor(obj):
 
     method_descriptor = type(list.append)
 
-    try:
+    if name in ATTR_MAP:
         attr_category = ATTR_MAP[name]
         if isinstance(attr_category, list):
             for condition, category in attr_category:
                 if condition(obj):
                     return category
         return attr_category
```

Comment:
There's no guarantee that `attr_category` and the above `category` will always be a tuple. I suggest keep `get_attr_category` as it was(aka return tuple or single value), then deal with different cases in `check_slotted`.